### DISCUSSION TITLE
Filters do not work for statuses with quotation marks in them

### DISCUSSION THREAD

**Discussion by davidbyoung** [community/community#10459] [Projects and Issues]
If you add a status such as `Won't Do` and try to add a filter, eg `-status:"Won't Do"`, you'll get the error "The filter is invalid, due to an unmatched quotation mark".  I've tried escaping it via `-status:"Won\'t Do"` and `-status:"Won''t Do"`, but neither worked.

**Comment by ADTC** [comment]
Just ran into this issue for Milestones as well. It's not limited to status.

**Reply by ADTC** [reply]
I guess the only fix now is to rename it to not have quotation mark. @davidbyoung `Wont Do`, `Rejected` or `Nope` ?

**Reply by davidbyoung** [reply]
Correct, unfortunately
